server/app/modules/DurableJobModule.java

```diff
@@ -147,4 +158,23 @@ public DurableJobRegistry provideRecurringDurableJobRegistry(
 
     return durableJobRegistry;
   }
+
+  @Provides
+  @StartupJobsProviderName
+  public DurableJobRegistry provideStartupDurableJobRegistry() {
+    //  PersistedDurableJobRepository persistedDurableJobRepository) {
```

Comment:
**suggestion:** Can we put a todo here instead with a bug tracking this work?

Reply:
Typically I would, but I'm going to be adding the first job in the next day for the service area work I'm doing.